frontend/app/(profile)/profile/my-projects/page.tsx

```
@@ -0,0 +1,180 @@
+            </div>
+          ))}
+        </div>
+      ) : (
+        <div className="text-center py-10 bg-white rounded-lg shadow-sm border">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-16 w-16 mx-auto text-gray-400 mb-4"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)